pipeline_config.json

```diff
@@ -7,6 +7,10 @@
     "45939459-e4d5-4ad6-8650-3e7134b9a035"
   ],
   "RecoveryCSVURLs": ["gs://avf-project-datasets/2019/IMAQAL/recovered_hormuud_week_6.csv"],
+  "PhoneNumberUuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "IMAQAL"
```

Comment:
How painful is it to rename this to IMAQAL_phone_number_avf_phone_id?

If it's easy to re-run the migration, please do that, otherwise let's leave it for now and rename it when we name ADSS -> CSAP_phone_number_avf_phone_id

Reply:
Not at all letme rename and do the migration

Reply:
Unfortunately I can't rename the one on firebase so to avoid duplication I will keep it as is for now.